core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_MDRange.hpp

```diff
@@ -55,13 +58,13 @@ class ParallelFor<FunctorType, Kokkos::MDRangePolicy<Traits...>,
                   Policy::inner_direction == Iterate::Right);
 
     execute_tile<Policy::rank>(
-        unused, functor, policy,
+        unused, m_functor, policy,
         std::integral_constant<Iterate, Policy::inner_direction>());
   }
 
   template <int Rank>
   inline std::enable_if_t<Rank == 2> execute_tile(
-      typename Policy::point_type offset, const FunctorType& functor,
+      typename Policy::point_type offset, const FunctorAdapter& m_functor,
```

Comment:
```suggestion
      typename Policy::point_type offset, const FunctorAdapter& functor,
```
I would avoid shadowing the member variable.